Documentation/guides/building-apps/build-properties.md

```diff
@@ -549,6 +549,39 @@ The most common values for this property are:
 
 Added in Xamarin.Android 6.1.
 
+## AndroidIncludeWrapSh
+
+A boolean value which indicates whether the Android wrapper script
+([`wrap.sh`](https://developer.android.com/ndk/guides/wrap-script))
+should be packaged into the APK. The property defaults to `false`
+since the wrapper script may significantly influence the way the
+application starts up and works and the script should be included 
+only when necessary e.g. for debugging or otherwise changing the
+application startup/runtime behavior.
+
+The script is added to the project using the
+[`@(AndroidNativeLibrary)`](./build-items.md#androidnativelibrary)
+build action, because it is placed in the same directory where
+architecture-specific native libraries, and must be named `wrap.sh`.
+The name can be specified using the `Link` metadata of the
+`AndroidNativeLibrary`:
+
+```xml
+<AndroidNativeLibrary Include="/path/to/my/arm64-wrap.sh" Condition=" '$(UseASAN)' != '' ">
```

Comment:
"Semantic" question: for native libs, the ABI can be "deduced" via "path sniffing".  Is the same possible with `wrap.sh` as well, e.g. given:

```xml
<AndroidNativeLibrary Include="path/to/lib/arm64-v8a/wrap.sh" />
```

will the ABI be inferred as `arm64-v8a`, thus allowing us to *skip* the `%(AndroidNativeLibrary.Link)` metadata?

If not, can this be done?

Reply:
Yes, it will properly "sniff" the architecture, but since you **must** name the script `wrap.sh` you won't be able to store two different scripts in the same architecture's directory - this is when `Link` is needed (as it is the case in this PR)